rest/data/buildlogger.go

```diff
@@ -625,7 +631,16 @@ func (mc *MockConnector) FindGroupedLogs(ctx context.Context, opts BuildloggerOp
 func (mc *MockConnector) findLogsByTestName(ctx context.Context, opts *BuildloggerOptions) (dbModel.LogIterator, error) {
 	logs := []dbModel.Log{}
 	for _, log := range mc.CachedLogs {
-		if log.Info.TaskID == opts.TaskID && log.Info.TestName == opts.TestName && containsTags(opts.Tags, log.Info.Tags) {
+		if log.Info.TaskID == opts.TaskID && containsTags(opts.Tags, log.Info.Tags) {
+			if opts.Group != "" && !containsTags([]string{opts.Group}, log.Info.Tags) {
```

Comment:
Just to be clear - a group is equivalent to a single tag?

Reply:
Yes